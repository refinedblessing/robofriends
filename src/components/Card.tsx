import * as React from 'react';

interface CardStatelessProps {
  name: string;
  username: string;
  email: string;
}

const Card: React.FunctionComponent<CardStatelessProps> = ({name, email, username}):JSX.Element => {
  return (
    <div className='bg-light-green dib br3 ma2 grow bw2 shadow-5 tc'>
      <img src={`https://robohash.org/${username}/size=300x300`} alt='robots'/>
      <div>
        <h2>{name}</h2>
        <p>{email}</p>
      </div>
    </div>
  )
}

export default Card;
