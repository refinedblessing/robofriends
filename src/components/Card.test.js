import { shallow } from 'enzyme'
import Card from './Card';
import React from 'react'

it ('enzyme works', () => {
  expect(shallow(<Card/>).length).toEqual(1)
  expect(shallow(<Card/>)).toMatchSnapshot();
})