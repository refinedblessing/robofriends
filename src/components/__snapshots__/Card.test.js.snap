// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`enzyme works 1`] = `
<div
  className="bg-light-green dib br3 ma2 grow bw2 shadow-5 tc"
>
  <img
    alt="robots"
    src="https://robohash.org/undefined/size=300x300"
  />
  <div>
    <h2 />
    <p />
  </div>
</div>
`;
